spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -680,12 +680,31 @@ If the `Acknowledgment` annotation is not set, the default policy is applied.
 
 IMPORTANT: Method only annotated with `@Outgoing` do not support acknowledgement as they don't receive an input `Message`.
 
+When a method annotated with `@Incoming` defines its acknowledgement policy to `PRE_PROCESSING` or `POST_PROCESSING`, the Reactive Messaging implementation is responsible of the acknowledgement of the message.
```

Comment:
When a method annotated with `@Incoming` defines its acknowledgement policy to *be*`PRE_PROCESSING` or `POST_PROCESSING`, the Reactive Messaging implementation is responsible *for* the acknowledgement of the message.